experimental/packages/opentelemetry-instrumentation-http/test/functionals/http-enable.test.ts

```diff
@@ -570,6 +570,7 @@ describe('HttpInstrumentation', () => {
             },
             component: 'http',
             noNetPeer: true,
+            error: err,
```

Comment:
This test asserts the error for "outgoing" request.
I would expect that we need to assert it also in some "incoming" request, but tests are green 🤔

Reply:
Could it be because it's an optional parameter?

Reply:
Since it is optional, where we don't set the value it becomes `undefined`, which is the asserted in `assertSpan` with `assert.strictEqual(span.events.length, 0);`.

I guess these tests are just missing, but the PR is fine as is (you are of course welcome to add them if you like)